examples/data_source/main.tf

```
@@ -13,20 +13,27 @@ provider "aap" {
   insecure_skip_verify = true
 }
 
-variable "state_id" {
-  type = number
-  description = "The id of the state to request"
+resource "aap_inventory" "my_inventory" {
+  name = "My new inventory"
+  description = "A new inventory for testing"
+  variables = jsonencode(
+    {
+      "foo": "bar"
+    }
+  )
```

```suggestion
variable "inventory_id" {
   type = number
   description = "The inventory id"
}
```

It is a best practice to keep only the `aap_inventory` data source in this example file as this is one being explained here